src/train_ner/train.py

```
@@ -0,0 +1,61 @@
+    )
+
+    X_train, X_val, y_train, y_val = train_test_split(texts, tags, test_size=0.2)
+
+    train_data = tokenize_and_align_labels(tokenizer, X_train, y_train, label2id)
+    val_data = tokenize_and_align_labels(tokenizer, X_val, y_val, label2id)
+
+    train_dataset = Dataset.from_dict(train_data)
+    val_dataset = Dataset.from_dict(val_data)
+
+    training_args = TrainingArguments(
+    output_dir="./ner_model",
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)